### DISCUSSION TITLE
Make video stream that supports the Wii U internet browser

### DISCUSSION THREAD

**Discussion by thepwrtank18** [jellyfin/jellyfin#9677] [Q&A]
I'm trying to make:
* 1080p max resolution
* mp4 container
* H264 - MPEG-4 AVC (part 10) (avc1) video codec
* MPEG AAC Audio (mp4a) audio codec, stereo

I tried /Videos/[id]/stream?api_key=[key]&width=1920&height=1080&container=mp4&videoCodec=h264&audioCodec=aac&level=4.0&requireAvc=true, but it plays at 238p on VLC and not at all on Firefox. As stupid as it sounds, the main target is the Wii U internet browser, which definitely won't work with this if Firefox doesn't.

This is what a video that plays properly looks like:
![image](https://user-images.githubusercontent.com/57580668/233879258-98c7e16c-f7f0-49bb-800f-196e1323b978.png)
This is the equivalent of the "Creator 1080p60" preset in Handbrake.

**Comment by Duoslow** [comment]
any update ?
